pkg/types/types.go

```diff
@@ -170,6 +171,10 @@ func NewWorkflowRunFromRaw(runRaw *github.WorkflowRun) *WorkflowRun {
 		run.Repository.Owner.Login, run.Repository.Name, run.ID,
 	)
 
+	if len(runRaw.PullRequests) > 0 {
+		run.BaseBranch = *runRaw.PullRequests[0].Base.Ref
```

Comment:
Combined with `ExcludePullRequests: false` from https://github.com/isovalent/corgi/pull/21/files#diff-429e584d53fae228e8f0c4857eb4d7dc07e9d3cdba33a2f924125c95ff8ef787R90 we can get the base branch against which the PR was opened, if it was a PR.